app/assets/stylesheets/application.css.scss

```diff
@@ -6,3 +6,16 @@
 @import "daplog/base";
 @import "daplog/blocks";
 @import "daplog/contexts";
+
+#foo{}
+a.Bar   {
+    margin : 0 0 0;
+	backgrounds:url(https://example.com/assets/image.png) ;
+  & > { color: red; }
+  content: 'foo'! important;
+  -webkit-border-radius: 0px .500em;
+  border: none solid rgba(0,0,0,0); /* baz */
```

Comment:
Use `//` comments everywhere<br>Commas in function arguments should be followed by a single space